src/server/scripts/Northrend/ChamberOfAspects/RubySanctum/boss_halion.cpp

```diff
@@ -889,52 +873,74 @@ class npc_orb_carrier : public CreatureScript
         struct npc_orb_carrierAI : public ScriptedAI
         {
             npc_orb_carrierAI(Creature* creature) : ScriptedAI(creature),
-                instance(creature->GetInstanceScript())
+                _instance(creature->GetInstanceScript())
             {
                 ASSERT(creature->GetVehicleKit());
+                me->SetDisplayId(me->GetCreatureTemplate()->Modelid1);
```

Comment:
? It should have the invisible trigger extra flag, so that shouldnt be needed ? You do see it in .gm on but that's intended